tests/components/ohme/test_sensor.py

```diff
@@ -0,0 +1,51 @@
+"""Tests for sensors."""
+
+from datetime import timedelta
+from unittest.mock import MagicMock, patch
+
+from freezegun.api import FrozenDateTimeFactory
+from ohme import ApiException
+from syrupy import SnapshotAssertion
+
+from homeassistant.const import STATE_UNAVAILABLE, Platform
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers import entity_registry as er
+
+from . import setup_integration
+
+from tests.common import MockConfigEntry, async_fire_time_changed, snapshot_platform
+
+
+async def test_sensors(
+    hass: HomeAssistant,
+    entity_registry: er.EntityRegistry,
+    snapshot: SnapshotAssertion,
+    mock_config_entry: MockConfigEntry,
+) -> None:
+    """Test the Ohme sensors."""
+    with patch("homeassistant.components.ohme.PLATFORMS", [Platform.SENSOR]):
+        await setup_integration(hass, mock_config_entry)
+
+    await snapshot_platform(hass, entity_registry, snapshot, mock_config_entry.entry_id)
+
+
+async def test_sensors_unavailable(
+    hass: HomeAssistant,
+    freezer: FrozenDateTimeFactory,
+    mock_config_entry: MockConfigEntry,
+    mock_client: MagicMock,
+) -> None:
+    """Test that sensors show as unavailable after a coordinator failure."""
+    with patch("homeassistant.components.ohme.PLATFORMS", [Platform.SENSOR]):
+        await setup_integration(hass, mock_config_entry)
```

Comment:
We don't have to limit the platforms here

Reply:
Will it be needed when more platforms are added?

Reply:
Nope

Reply:
Ah - my mistake. Remove on line 39 but leave on line 26?

Reply:
Yes, because it is needed for snapshot_platforms